api/v1beta1/condition_consts.go

```diff
@@ -251,3 +251,22 @@ const (
 	// ScalingDownReason (Severity=Info) documents a MachineSet is decreasing the number of replicas.
 	ScalingDownReason = "ScalingDown"
 )
+
+// Conditions and condition reasons for Cluter Topology.
+const (
+	// TopologyReconciledCondition defines the Recondiled condition type that summarizes the reconciliation state of
+	// a Cluster Topology.
+	TopologyReconciledCondition ConditionType = "TopologyReconciled"
+
+	// TopologyReconciledErrorReadon (Severity=Error) documents a condition not in Status=False because the
+	// topology reconciler errored.
+	TopologyReconciledErroredReason = "TopologyReconcileErrored"
+
+	// TopologyReconciledControlPlaneUpdatePendingReason (Severity=Info) documents a condition in Status=False because the
+	// associated Control Plane is not yet updated to match the desried topology.
+	TopologyReconciledControlPlaneUpdatePendingReason = "ControlPlaneUpdatePending"
```

Comment:
A thought: "not yet updated to match the desired spec." <- this could be confusing on first glance as it could imply that underlying control plane object (real object) is not yet update to the desired spec of the control plane. However this is not what the condition reflects. 
How about "not yet updated to match the desired topology spec."? <- this makes it explicit that the desired spec here is the topology spec and not the control plane spec.